src/clients/node/package.json

```diff
@@ -36,7 +34,7 @@
   "scripts": {
     "benchmark": "./scripts/benchmark.sh",
     "test": "node dist/test",
-    "postinstall": "npm run install_zig && npm run download_node_headers && npm run build_lib",
+    "postinstall": "test -d src/tigerbeetle || (mkdir -p src/tigerbeetle && cp -r ../../../{scripts,src/{*.zig,io,lsm,testing,vsr,state_machine}} src/tigerbeetle) && npm run install_zig && npm run download_node_headers && npm run build_lib",
```

Comment:
If you put this in a script we can shellcheck it. Shellcheck will also validate that all the commands and syntax you used work in posix sh (alpine doesn't have bash, nor do we require it).

Reply:
But you'd need to add that file to the `files` list in package.json.

Reply:
The alpine build failure is likely because of bashisms

Reply:
Yeah, this in generall going to be a bite more complicated/ugly than I thought. Turns out I can't use `prepare` hook for this, cause it runs after install. 

So I think what we could do is indeed put the logic into a script, run it on postinstall, but make the script _itself_ conditional such that it only does the copy if in the repo...

Reply:
Yeah, busybox/posix shell doesn't support `{a,b,c}`.